codegen/templates/resource.go.tpl

```diff
@@ -168,8 +168,8 @@ func update{{ .UpperCaseDifferentiator }}Resource(d *schema.ResourceData, meta i
 	{{- range .Parameters }}
 	{{- if not .IsPathParam }}
 	  {{- if not .Computed }}
-	if d.HasChange("{{ .Name }}") {
-		data["{{ .Name }}"] = d.Get("{{ .Name }}")
+	if raw, ok := d.GetOk("{{ .Name }}"); ok {
```

Comment:
This edit was due to a bug I found in the template when testing. If I sent up a field again and it hadn't changed, it wasn't passed through to Vault. This fix ensures that if it's been sent, it's passed through.